docs/02.API-REFERENCE.md

```diff
@@ -10440,108 +10386,16 @@ main (void)
   static uint32_t global_mode_snapshot_buffer[256];
   const jerry_char_t script_to_snapshot[] = "(function () { return 'string from snapshot'; }) ();";
 
-  jerry_value_t generate_result;
-  generate_result = jerry_generate_snapshot (script_to_snapshot,
-                                             sizeof (script_to_snapshot) - 1,
-                                             NULL,
-                                             0,
-                                             global_mode_snapshot_buffer,
-                                             sizeof (global_mode_snapshot_buffer) / sizeof (uint32_t));
-
-  if (!jerry_value_is_error (generate_result))
-  {
-    size_t snapshot_size = (size_t) jerry_get_number_value (generate_result);
-  }
-
-  jerry_release_value (generate_result);
-
-  jerry_cleanup ();
-  return 0;
-}
-```
-
-**See also**
-
-- [jerry_init](#jerry_init)
-- [jerry_cleanup](#jerry_cleanup)
-- [jerry_generate_function_snapshot](#jerry_generate_function_snapshot)
-- [jerry_exec_snapshot](#jerry_exec_snapshot)
-- [jerry_parse_options_t](#jerry_parse_options_t)
-
-
-## jerry_generate_function_snapshot
```

Comment:
We should mention that using the `jerry_generate_snapshot` method the same functionality (which was provided by this removed function) can be achieved/reproduced. Ideally an example should also be there.

Reply:
I am not sure we need to overdo the examples.